jdaviz/app.py

```diff
@@ -1013,21 +1013,18 @@ def add_data_to_viewer(self, viewer_reference, data_path,
         viewer_reference : str
             The reference to the viewer defined with the ``reference`` key
             in the yaml configuration file.
-        data_path : str
-            Either the data filename or the Glue data label found in the ``DataCollection``.
+        data_label : str
+            The Glue data label found in the ``DataCollection``.
         clear_other_data : bool
             Removes all other currently plotted data and only shows the newly
             defined data set.
-        ext: str
-            The data extension to access from a file.  If data_path is a filename, ext
-            is required.
         """
         viewer_item = self._viewer_item_by_reference(viewer_reference)
         if viewer_item is None:  # Maybe they mean the ID
             viewer_item = self._viewer_item_by_id(viewer_reference)
         if viewer_item is None:
             raise ValueError(f"Could not identify viewer with reference {viewer_reference}")
-        data_label = self.return_data_label(data_path, ext=ext, check_unique=False)
```

Comment:
what is this line being removed?  This used to be a `data_label = self._build_data_label(data_path, ext=ext)`, which I still see in my code, as of 3.0.1?  Does the static method `_build_data_label` still exist?

Reply:
Ahh, I missed that. thanks!